src/components/box/Box.tsx

```diff
@@ -13,6 +13,7 @@ type BoxProps = {
   backgroundColor?: string;
   radius?: "small" | "medium" | "large";
   spacing?:
+    | "spacing00"
```

Comment:
GNUI Figma says nothing about "spacing0". This creates a mess.

Reply:
ok i have changed it